localstack/testing/snapshots/transformer.py

```diff
@@ -1,13 +1,14 @@
 import copy
 import logging
+import os
 import re
 from re import Pattern
 from typing import Callable, Optional, Protocol
 
 from jsonpath_ng.ext import parse
 
-LOG = logging.getLogger(__name__)
-
+SNAPSHOT_LOGGER = logging.getLogger(__name__)
+SNAPSHOT_LOGGER.setLevel(logging.DEBUG if os.environ.get("SNAPSHOT_DEBUG") else logging.WARNING)
```

Comment:
i think `SNAPSHOT_DEBUG` is better than `DEBUG_SNAPSHOT` (since the first specifies the namepsace of the property), however we already have `DEBUG_ANALYTICS`, `DEBUG_PLUGINS`, so for the sake of consistency it would maybe be good to rename the variable.